torch/_dynamo/config.py

```diff
@@ -485,12 +482,14 @@ def default_debug_dir_root():
     "storage_offset",
     "layout",
     "data",
+    "is_neg",
+    "is_nonzero",
+    "is_pinned",
+    "is_set_to",
+    "is_shared",
+    "is_signed",
```

Comment:
Let's just do the following:
- is_neg
- is_conj
- is_pinned

Also, we could allow stride now (maybe in a follow-up...) because it'll return the unbacked SymInts :).